Octokit/Helpers/AcceptHeaders.cs

```diff
@@ -15,5 +15,7 @@ public static class AcceptHeaders
         public const string StarCreationTimestamps = "application/vnd.github.v3.star+json";
 
         public const string CommitReferenceSha1Preview = "application/vnd.github.chitauri-preview+sha";
+
+        public const string SquashCommitPreview = "application/vnd.github.polaris-preview+json";
```

Comment:
I'm not seeing where we use this header in the client itself. This should do something inside `PullRequestClient.Merge(..., MergePullRequest)`